mw/lib/reverts/detector.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, radius=defaults.RADIUS):
         """
         :Parameters:
             radius : int
-                the maximum revision distance that a revert can span.
+                a positive integer indicating the maximum revision distance that a revert can span.
         """
         super().__init__(maxlen=radius + 1)
```

Comment:
This would be a good chance to check that the radius is positive and throw a TypeError otherwise.